kong/router/atc.lua

```diff
@@ -64,6 +65,7 @@ do
                   "http.method", "http.host",
                   "http.path", "http.raw_path",
```

Comment:
I found a bug in here. Could you fix it as well in the same PR @chronolaw :

See: KAG-2289

Reply:
removed.